db/1.10/modules/ROOT/pages/database-connector-select.adoc

```diff
@@ -107,6 +107,20 @@ In the XML editor, the `<db:sql>` configuration looks like this:
 
 You can apply input parameters only to parameters in a `WHERE` clause. To modify any other part of the query, use the DataWeave interpolation operator.
 
+Since the expression values may not be available at design time, using this feature might throw DataWeave evaluation errors or affect the Output Metadata (DataSense) of the Select Operation.
```

Comment:
```suggestion
Because the expression values may not be available at design time, using dynamic queries might throw DataWeave evaluation errors or affect the output metadata (DataSense) of the *Select* operation.
```